lib/aws/lex/conversation/slot/elicitation.rb

```diff
@@ -53,9 +53,7 @@ def increment_slot_elicitations!
           end
 
           def maximum_elicitations?
-            return false if maximum_elicitations.zero?
-
-            elicitation_attempts > maximum_elicitations
+            elicitation_attempts >= maximum_elicitations
```

Comment:
Just a thought - if we're already making a breaking change:

What do you think about renaming `maximum_elicitations -> max_retries`? That matches the Lex terminology.

Reply:
I changed it to max_retries